### ISSUE TITLE
Window Frame Reduction | Clarification

### ISSUE THREAD

**Issue by pedrammrbn** [architecture-building-systems/hive#818]
In the code, there is a commented-out variable f_sh for "Reduktion Solare Waermeeintraege" which is equal to 0.9 but is not used in the solar gain computations, or anywhere else for that matter:

![image](https://github.com/architecture-building-systems/hive/assets/108935429/3ac88a53-6dd1-4e5f-98e0-716651d43777)

The window areas are directly read off of the building inputs, and not modified. As shown in the output from the <SIA 2024 Room Reader>component:

![image](https://github.com/architecture-building-systems/hive/assets/108935429/e00aa2bd-d6d5-4500-be12-909d1fa06183)

And then the input to the <SIA 380> solver component:

![image](https://github.com/architecture-building-systems/hive/assets/108935429/b1234503-b79a-4e73-ad63-517d3ecf1ee8)

This was further tested by changing the value of the Hive.IO.Building.Building.WindowReductionFactor in the code which resulted in no change in simulation results.

**ADDITIONALLY:** According to SIA 380/1:2016 there is a separate 0.9 reduction factor for solar gains AND another factor for window frame reduction that defaults to 0.75 if undefined.  (SIA 380.1 - 3.5.4.12)
These two factors are shown in (SIA 380.1 - P.54 (116))

![image](https://github.com/architecture-building-systems/hive/assets/108935429/d9e2e682-84fc-42ff-94fb-b318d5d1bd50)

![image](https://github.com/architecture-building-systems/hive/assets/108935429/a574e001-447b-4c93-9f50-3d16723ec7b9)

![image](https://github.com/architecture-building-systems/hive/assets/108935429/0c09a8f7-218a-436a-9fd4-3475ee4a3ff9)

So in conclusion? Using the modeled Q_solar provides more accuracy, but perhaps a frame reduction factor should be implemented to account for the difference in window areas.

**Comment by christophwaibel** [comment]
thanks for the summary! Yes, the conclusion is that we should treat it similar to how we have the floor area reduction: do it somewhere in the Hive building component.
Will do it asap
